test/cql-pytest/run.py

```diff
@@ -260,6 +261,13 @@ def check_ssl_cql(ip):
     ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
     check_cql(ip, ssl_context)
 
+## Test that the Scylla REST API is serving, for wait_for_services() below.
+def check_rest_api(ip, port=10000):
+    resp = requests.get(f"http://{ip}:{port}/ui/")
+    if resp.status_code != requests.codes.ok:
+        raise NotYetUp
```

Comment:
@nyh did you check that?
What exception did you get?

Reply:
No, I didn't check it (to check this you should probably remove the wait for cql and leave only the wait for rest api). I believe that requests.get throws a ConnectionError if the host is not available - it doesn't return an http status_code because there is none.

Reply:
you're right, will fix.